src/sass/App.scss

```diff
@@ -1,15 +1,17 @@
 $fa-font-path: "~font-awesome/fonts";
-
 @import "./_variables.scss";
+
 @import "../../node_modules/@edx/edx-bootstrap/sass/edx/theme";
 @import "../../node_modules/bootstrap/scss/bootstrap";
 @import "../../node_modules/@edx/paragon/src/index";
 @import "../../node_modules/font-awesome/scss/font-awesome";
 @import "../../node_modules/@edx/frontend-component-footer/src/lib/scss/site-footer";
-@import "../components/RichEditor/RichEditor.scss";
 // Required to have the Collapsible component work
 @import '../../node_modules/@edx/paragon/src/Collapsible/Collapsible';
```

Comment:
The two lines above were removed by me in a prevoius PR - looks like you added it back in one of your merges? Can you drop them here again, while you're there?

Also, all of our imports are now relative with the `../../node_modules`? They should just be doing `~` or relative to `node_modules` and our webpack config should look in `node_modules` for them... Looks like a change you made in a previous PR? I guess you don't need to clean that bit up here, but it's not our preferred import strategy.